### PR TITLE
fix: docker image build (#1437)

### PR DESCRIPTION
**Pull request by tkrop**
Finally fixes the docker build problem.

### CODE REVIEW

**Comment by tkrop** [comment]
👍

File: delivery.yaml [hunk 1]
```diff
@@ -28,13 +29,7 @@ pipeline:
     commands:
       - desc: Build Web UI
         cmd: |
-          cd $PWD/web-ui
-          yarn install -q ; yarn test ; yarn build
-          if [ -n "${CDP_PULL_REQUEST_NUMBER}" ]; then
-            echo "We're in a pull request, aborting."; exit 0;
-          fi;
-          PREFIX="pierone.stups.zalan.do/builder-knowledge"
-          IMAGE="${PREFIX}/zally-web-ui-dummy:${CDP_BUILD_VERSION}"
```
Comment (kgalli) [inline delivery.yaml:37]: We do not have to build the docker image anymore?

Comment (tkrop) [reply delivery.yaml:37]: No we do not need this any more. Zally-Web-UI is retired. This image is not accessible for the public, so we can stop deploying it.

**Comment by natalyapeixoto** [comment]
👍

File: delivery.yaml (hunk 1 above)
Comment (kgalli) [reply delivery.yaml:37]: Why do we even build this step then? Is it to ensure it can be build?

**Pull request merged**